services/regulated-assets-approval-server/internal/serve/api_kyc_status_test.go

```diff
@@ -278,6 +309,42 @@ func TestAPI_GETKYCStatus(t *testing.T) {
 	require.NotNil(t, kycStatusGETResponseApprove.ApprovedAt)
 	require.Nil(t, kycStatusGETResponseApprove.RejectedAt)
 
+	// INSERT new account in db's accounts_kyc_status table; new account was rejected after submiting kyc.
```

Comment:
if I'm reading this right:
```suggestion
	// INSERT new account in db's accounts_kyc_status table; new account was pending after submiting kyc.
```

Reply:
https://github.com/stellar/go/pull/3673/commits/1dfa80f03740c97be3edd1fa99eaeae092beb78f